mteb/tasks/Retrieval/deu/GerDaLIRRetrieval.py

```diff
@@ -25,8 +25,8 @@ class GerDaLIR(AbsTaskRetrieval):
         eval_langs=["deu-Latn"],
         main_score="ndcg_at_10",
         date=None,
-        domains=None,
-        task_subtypes=None,
+        domains=["Legal"],
+        task_subtypes=["Legal information retrieval"],
```

Comment:
```suggestion
        task_subtypes=[],
```
I would say it is just Domain + task-type rather that a subspecialization of task type